example/src/main/java/org/aerogear/mobile/example/ExampleApplication.java

```diff
@@ -1,11 +1,16 @@
 package org.aerogear.mobile.example;
 
+import java.text.MessageFormat;
+
 import android.app.Application;
+import android.widget.Toast;
 
 import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.http.HttpResponse;
+import org.aerogear.mobile.core.metrics.MetricsPublisherListener;
 import org.aerogear.mobile.core.metrics.MetricsService;
 
-public class ExampleApplication extends Application {
+public class ExampleApplication extends Application implements MetricsPublisherListener {
 
     private MobileCore mobileCore;
```

Comment:
Are these meant to be reinitialized? If that's not the case, they should be final